### PR TITLE
HPCC-13212 - sendlimit test failing on Roxie when localSlave set

### PR DESCRIPTION
**Pull request by richardkchapman**
Signed-off-by: Richard Chapman rchapman@hpccsystems.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@AttilaVamos @ghalliday Please review. This is safe enough to include in gold.

**Comment by AttilaVamos** [comment]
I can't see any differences in OBT.
I cerated a sendlimit2.ecl with your changes but it generates same output than the previous version. 
What is the expected output of this test? It is differ from  the previous version:
<Row><x>Packet length</x><value>1446</value></Row> 

I can get: 

ecl run -t roxie ecl/sendlimit2.ecl

<Result>
<Dataset name='Result 1'>
 <Row><x>a</x><value>1</value></Row>
 <Row><x>b</x><value>2</value></Row>
 <Row><x>c</x><value>3</value></Row>
 <Row><x>d</x><value>4</value></Row>
</Dataset>
</Result>

**Comment by AttilaVamos** [comment]
Yes, this solution pass on OBT.

File: testing/regress/ecl/sendlimit.ecl [hunk 1]
```diff
@@ -15,11 +15,17 @@
...
 ############################################################################## */
 
-//This is primarily here to ensure that roxie fails gracefully when a packet that is too large is sent from the master to the slave
+// This is here to ensure that roxie fails gracefully when a packet that is too large
+// is sent from the master to the slave. Note that the limit is not applied when localSlave is set,
+// so the test is meaningless - in order to ensure that the results still match on such systems,
+// we simply fake the expected output.
+
 //nothor
 //nothorlcr
```
Comment (ghalliday) [inline testing/regress/ecl/sendlimit.ecl]: trivial: only one of these is needed

**Pull request merged**